engine/api/scheduler/purge.go

```diff
@@ -0,0 +1,15 @@
+package scheduler
+
+import (
+	"github.com/go-gorp/gorp"
+
+	"github.com/ovh/cds/engine/api/workflow"
+	"github.com/ovh/cds/sdk/log"
+)
+
+// PurgeRun to purge all history of workflow run
+func PurgeRun(db *gorp.DbMap) error {
+	log.Debug("PurgeRun> Deleting all workflow run marked to delete...")
```

Comment:
This should be in the scheduler package. The scheduler package is for pipeline scheduler.
I think this have to be moved to the workflow package